src/ImageSharp/Formats/Jpeg/Components/Encoder/RgbToYCbCrConverterLut.cs

```diff
@@ -63,10 +64,10 @@ internal unsafe struct RgbToYCbCrTables
         /// <summary>
         /// Initializes the YCbCr tables
         /// </summary>
-        /// <returns>The initialized <see cref="RgbToYCbCrTables"/></returns>
-        public static RgbToYCbCrTables Create()
+        /// <returns>The initialized <see cref="RgbToYCbCrConverterLut"/></returns>
+        public static RgbToYCbCrConverterLut Create()
```

Comment:
Is this masked by a condition now? I can't spot anything in the code. It would be good to avoid it if possible.

Reply:
`Create` is only called in `YCbCrForwardConverter<TPixel>` if the Vectorized variant isn't available. One leftover would be that the space for the tables is still allocated on the stack even if it is never used.